DTS-STN/secure-client-hub
# 113144 dynamic routes

## [ADO-113144](https://dev.azure.com/VP-BD/DECD/_workitems/edit/113144)

### Description
This PR updates the contact us pages to use dynamic routing instead of having an individual file for each page, given they follow the same format. This will allow us to easily add more contact pages in the future (ie. SINOM) and also cuts down on quite a bit of code. I've consolidated the three mappers into a single mapper that searches through the list of contact pages based on the id coming from AEM.

Fixes [AB#113144](https://dev.azure.com/VP-BD/DECD/_workitems/edit/113144)

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/contact-us`
4. Click each of the links and ensure the proper content is shown (both in EN and FR)
